label_studio/core/migrations/0001_initial.py

```diff
@@ -1,7 +1,7 @@
 # Generated by Django 3.2.13 on 2022-06-27 01:16
```

Comment:
file-level comment - imo, we should exclude migrations from all linting / autofixes. This should be configured in ruff.toml.